src/harness/unittests/compileOnSave.ts

```diff
@@ -588,5 +588,37 @@ namespace ts.projectSystem {
             assert.isTrue(outFileContent.indexOf(file2.content) === -1);
             assert.isTrue(outFileContent.indexOf(file3.content) === -1);
         });
+
+        it("should emit js files in inferred projects when they could be input files", () => {
+            const f1 = {
+                path: "/a/b/f1.ts",
+                content: "var test = \"\";"

```

Comment:
Nit: Prefer single quotes for a string that contains double quotes. (Or just use `var test = 0;`.)